frontend/src/app/components/transaction/transaction.component.html

```diff
@@ -22,7 +22,7 @@ <h1 i18n="shared.transaction">Transaction</h1>
         <app-clipboard [text]="txId"></app-clipboard>
       </div>
 
-      <div class="container-buttons float-right">
+      <div class="container-buttons">
```

Comment:
Add text-right to classes for and RTL hinting.

Reply:
I've solve this by adding the classes under the `.rtl-layout`.